android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -20,6 +20,8 @@ import com.revenuecat.purchases.identifyWith
 import com.revenuecat.purchases.purchasePackageWith
 import com.revenuecat.purchases.purchaseProductWith
 import com.revenuecat.purchases.resetWith
+import com.revenuecat.purchases.logInWith
```

Comment:
from this point on, the changes had been there merged before, but were removed in #75, because identity v3 wasn't ready to go in the native SDKs